apis/frontend/oss/README.md

```diff
@@ -3,10 +3,10 @@
 Nosto offers various Open Source libraries for integration of Nosto products into frontend app development.
 The libraries are hosted on GitHub.
 
-| Library | Description | API Docs |
+| Library | Description | Links |
 | :-- | :-- | :-- |
-| [nosto-js](https://github.com/Nosto/nosto-js) | Nosto JS is a wrapper for the client script, providing an easy way to interact with the Nosto service from your JavaScript or TypeScript applications. Use this library for basic Nosto integration in FE app builds. | [API Docs](https://nosto.github.io/nosto-js/) |
-| [search-js](https://github.com/Nosto/search-js) | Search JS is a wrapper for the Nosto Search functionality. Use this library for Nosto Search usage in FE app builds. | [API Docs](https://nosto.github.io/search-js/) |
-| [nosto-autocomplete](https://github.com/Nosto/nosto-autocomplete) | Standalone library for Autocomplete implementations using Nosto Search. Use this library for Autocomplete implementations using Nosto Search in FE app builds. | [API Docs](https://nosto.github.io/nosto-autocomplete) |
-| [nosto-react](https://github.com/Nosto/nosto-react) | Nosto React is a React component library to make it easier to implement Nosto using the Session API. Use this library for embedding Nosto recommendations in a React app. | [API Docs](https://nosto.github.io/nosto-react/) |
-| [shopify-hydrogen](https://github.com/Nosto/shopify-hydrogen) | Component library to make it easier to integrate Nosto features into a Shopify Hydrogen-based storefront. | N/A |
\ No newline at end of file
+| [nosto-js](https://docs.nosto.com/techdocs/apis/frontend/oss/nosto-js) | Nosto JS is a wrapper for the client script, providing an easy way to interact with the Nosto service from your JavaScript or TypeScript applications. Use this library for basic Nosto integration in FE app builds. | [GitHub](https://github.com/Nosto/nosto-js) [API Docs](https://nosto.github.io/nosto-js/) |
```

Comment:
Can we force new line?
[GitHub](https://github.com/Nosto/nosto-js) <br/> [API Docs](https://nosto.github.io/nosto-js/)